### ISSUE TITLE
[Clarification & Feedback] Quick Start 2 - 5. Grouping Data - "Add New Column"

### ISSUE THREAD

**Issue by bcyee1** [sigmacomputing/sigmaquickstarts#103]
## TL;DR
Specify that it is through the Grouped By column that you're adding a new column and a request to clarify/explain the difference.

## New User Feedback
So I had a moment of confusion in following along in Quick Start 2 which comes down to _where_ I was clicking "add new column" from. See Quick Start (Page 5 - Grouping Data)

![Screenshot 2023-07-14 at 4 06 17 PM](https://github.com/sigmacomputing/sigmaquickstarts/assets/139162552/9e61d928-b048-4550-b4be-a1bc019ff540)

> Add a new column and rename it `Monthly Profit` using the following formula:

So, earlier in the Quick Start (Page 4 - Calculated Columns) you learn about creating columns at the table level.  However in Step 5 the phrasing "Add a new column and rename it" makes me think that you're asking
 to make a column just like I did in step 4.

However, doing that and then moving it to "Group By" (which is called out in the step previous as an option to Group) doesn't make the same as adding a column next to Group By.  Or at least it doesn't if you drag and drop like:
![Screenshot 2023-07-14 at 4 17 28 PM](https://github.com/sigmacomputing/sigmaquickstarts/assets/139162552/e07318ec-88b4-4541-a24a-6600bd62a320)

And I'm unable to drag/drop [Profit] into the Group By month grouping anyways.

## The Ask
1. Update the Quick Start to make it clearer that you need to use the "Add a column" from the Group By menu
2. Help users understand if they made a mistake, like "Does your data look like this?" Here's an explanation of what went wrong:
![Screenshot 2023-07-14 at 4 20 15 PM](https://github.com/sigmacomputing/sigmaquickstarts/assets/139162552/f9cf3298-00ee-482b-80ca-1eedcf0d8ba9)

**Comment by pballai** [comment]
I will add some instructional text to help explain this but the point you are missing is that we are not only adding a new column in this flow, we are also adding the column as part of a grouped columns calculation. This results in the calculation to be aggregated at the grouped columns level. This is a subtle, but important difference than just adding a table column that has some calculations.
